pkg/sql/sem/builtins/geo_builtins.go

```diff
@@ -1161,6 +1161,50 @@ var geoBuiltins = map[string]builtinDefinition{
 		},
 	),
 
+	"st_pointfromgeohash": makeBuiltin(
+		defProps(),
+		tree.Overload{
+			Types: tree.ArgTypes{
+				{"geohash", types.String},
+				{"precision", types.Int},
+			},
+			ReturnType: tree.FixedReturnType(types.Geometry),
+			Fn: func(ctx *tree.EvalContext, args tree.Datums) (tree.Datum, error) {
+				g := args[0].(*tree.DString)
+				p := args[1].(*tree.DInt)
+				ret, err := geo.GeohashToPoint(string(*g), int(*p))
+				if err != nil {
+					return nil, err
+				}
+				return tree.NewDGeometry(ret), nil
+			},
+			Info: infoBuilder{
+				info: "Return a lon/lat representation of geohash with the supplied precision.",
```

Comment:
nit: spell "geohash" as "GeoHash".
Also rephrase as "Return a Geometry point from a GeoHash string." -- I think this sentence is clearer :)

Reply:
can i rephrase as "Return a Geometry point from a GeoHash string with supplied precision." as this function also takes precision.

Reply:
done, let me know if we need to rephrase it.